pythran/dist.py

```diff
@@ -20,6 +55,9 @@ class PythranExtension(Extension):
     They must be processable by pythran, of course.
 
     The compilation process ends up in a native Python module.
+
+    Unlike distutils.core.Extension it also accepts a cxx keyword argument to
+    force the compiler executable.
```

Comment:
This is a good feature. If only it were implemented upstream, we could have avoided all these hacks :)